WaterDock-2.0_Basics.md

```diff
@@ -0,0 +1,76 @@
+**This file is a more detailed version of README.md.**

```

Comment:
For headings, rather than using **title** bolding, I'd  recommend using the # sign

example:
# Title

Reply:
Also do check out how it renders on github, you can see it here: https://github.com/guidodee/WaterDock-2.0/blob/master/WaterDock-2.0_Basics.md

As you can see, the title is on the same line as the following text.

Ideally I'd suggest having a bigger title that says "Waterdock Detailed Instructions", and then have the "this file is a more detailed version of README.md" as standard text.

Reply:
I agree, looks a lot better - will make those changes.